backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -436,12 +444,19 @@ protected Map<String, Pair<String, String>> getSharedLocks() {
             // StartVmBackup should handle locks
             return locks;
         }
-        if (!Guid.isNullOrEmpty(getParameters().getImageId()) && getDiskImage() != null) {
+
+        if (!Guid.isNullOrEmpty(getParameters().getImageId()) ||
+                getDiskImage() != null) {
```

Comment:
do we need this change? (we have to check `getDiskImage() != null`, otherwise we'll get NPE in the following line)